src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -384,59 +368,49 @@ private IOperation VisitShortCircuitingOperator(IBinaryOperation condition, bool
             // stopValue  |     0         1
 
             SpillEvalStack();
+            int captureId = captureIdForResult ?? _availableCaptureId++;
 
-            BasicBlock lazyFallThrough = null;
+            BasicBlock lazyFallThrough = stopSense ? fallToTrueOpt : fallToFalseOpt;
```

Comment:
Perhaps `ref BasicBlock lazyFallThrough = ref stopSense ? ref fallToTrueOpt : ref fallToFalseOpt;`, to avoid the need to assign back to `fallToTrueOpt` and `fallToFalseOpt` below.

Reply:
> Perhaps ref BasicBlock lazyFallThrough = ref stopSense ? ref fallToTrueOpt : ref fallToFalseOpt;, to avoid the need to assign back to fallToTrueOpt and fallToFalseOpt below.

Sure, coming in the next PR.

---
In reply to: [163283952](https://github.com/dotnet/roslyn/pull/24358#discussion_r163283952) [](ancestors = 163283952)